cmd/pauseloop/main.go

```diff
@@ -0,0 +1,48 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+Source:
+https://github.com/kubernetes-sigs/windows-testing/blob/master/images/pause/pauseloop.go
```

Comment:
this seems to have gone away some where, any idea it's new location?

Reply:
... from the current experience of the last months I did not need pauseloop at all.
That's also why I didn't try to trackdown the above link.